spk/sickbeard-custom/src/installer.sh

```diff
@@ -88,19 +92,21 @@ preuninst ()
 {
     # Stop the package
     ${SSS} stop > /dev/null
-
+    
     # Remove the user if uninstalling
-    if [ "${SYNOPKG_PKG_STATUS}" == "UNINSTALL" ]; then
-        syno_group_remove
+    
+	if [ "${SYNOPKG_PKG_STATUS}" == "UNINSTALL" ] && [ "${BUILDNUMBER}" -lt "7135" ]; then
```

Comment:
Please use spaces in shell files, no tabs throughout this file. 4 spaces, not 8, and watch the indents.